fairlearn/metrics/_fairness_metrics.py

```diff
@@ -54,7 +61,12 @@ def demographic_parity_difference(
 
 
 def demographic_parity_ratio(
-    y_true, y_pred, *, sensitive_features, method="between_groups", sample_weight=None
+    y_true,
+    y_pred,
+    *,
+    sensitive_features,
+    method: Literal["between_groups", "to_overall"] = "between_groups",
```

Comment:
It would be great to also update the related docstring below to reflect the change